gmongaras/MetaU_Capstone
# Adding more user controls

<b>What changed?</b>
Added a settings page which allows the user to change visibility of their account to friends and users who aren't their friends

<b>How to test</b>
- Change profile picture using the settings
- Log into a users account, click the menu in the upper right, and change some settings. Log in as another user and notice how the visibility of fortunes and/or the map changes based on the chosen settings
- Remove location access from the phone and then go into the user settings. Click "Allow location access" to manually give location access to the app.
- Toggle push notifications. If push notifications is on, the user should receive a push notification when a new fortune is ready. If the toggle button is off, the user should not receive a notification when a new fortune is ready.
- Delete the user's account using the button at the bottom. All friends and requests should be removed.
